pypeit/reduce.py

```diff
@@ -196,14 +190,19 @@ def parse_manual_dict(self, manual_dict, neg=False):
         # Return
         return manual_extract_dict
 
-    def find_objects(self, image, ivar, std=False, ir_redux=False, std_trace=None, maskslits=None,
+    def find_objects(self, std=False, ir_redux=False, std_trace=None, maskslits=None,
```

Comment:
Please return the original argument list.

Reply:
done.